src/main/java/org/icgc/argo/program_service/security/EgoRestSecurity.java

```diff
@@ -40,7 +41,7 @@
 
 @Slf4j
 @Service
-@Profile("auth")
+@Profile("default")

```

Comment:
This should still only run for `auth` profile

Reply:
changed profile to auth